Sources/Defaults/Defaults+Bridge.swift

```diff
@@ -337,6 +337,57 @@ extension Defaults {
 	}
 }
 
+extension Defaults {
+	public struct RangeBridge<T: RangeSerializable>: Bridge {
+		public typealias Value = T
+		public typealias Serializable = [Any]
+		typealias Bound = T.Bound
+
+		public func serialize(_ value: Value?) -> Serializable? {
+			guard let value = value else {
+				return nil
+			}
+			if Bound.isNativelySupportedType {
```

Comment:
Shouldn't this be handled in the `Bound.bridge.serialize` method implementation?

Reply:
`Bound.bridge` depends on `Bound` type. 
If `Bound.bridge` is a user custom bridge, we should not let them handle this. 
This situation seems to be similar to #101.

https://github.com/sindresorhus/Defaults/blob/3535f3d088113cf24705014eec6a17f0fd73237f/Sources/Defaults/Utilities.swift#L201-L211

`toSerializable` will only check once. I think we should make it recursively.

Reply:
> toSerializable will only check once. I think we should make it recursively.

👍